tests/sync/test_chooser_api.py

```
@@ -564,6 +565,150 @@ def test_chooser_view_triggers_prefill(self):
+        # Creating another skip with the same session key should fail
+        with self.assertRaises(Exception):
+            ChooserSkip.objects.create(
+                task=task,
+                session_key="test-session-123",
+            )
+
+
+class ChooserSkipAPITest(APITestCase):
+    """Test the POST /api/chooser/skip endpoint."""
+
+    fixtures = ["./fighthealthinsurance/fixtures/initial.yaml"]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)